### ISSUE TITLE
pre-train model on LOLv2

### ISSUE THREAD

**Issue by flyoutLMF** [JianghaiSCU/Diffusion-Low-Light#13]
I tested the PSNR and SSIM on the LOLv2 dataset with the pre-train model on LOLv1, it achieved the result reported in the paper. But the test set in LOLv2 and the train set of LOLv1 overlap significantly, so the result I got is not credible. Did you test your PSNR and SSIM on the LOLv2 dataset with the pre-train model trained on the LOLv1 dataset? If not, could you please offer the pre-train model of LOLv2? Thank you a lot!

**Comment by qiqi77z** [comment]
Have you solved this problem?

**Comment by qiqi77z** [comment]
Hello, may I ask how this file came from the test: LOLv1_train.txt？
